### ISSUE TITLE
Setup CI

### ISSUE THREAD

**Issue by lewis-fidlers** [borealisgroup/studio-blog#4]
So far Heroku has been setup so we have Pull request apps. I believe this to be a feature we'll need for other people to be interested in writing as well. 

Having people setup a dev environment just to preview a blog someone wrote is overkill and would probably demotivate when asking for a proofread.

However heroku is not a preferable platform to host static website, $7/m for a cert to work is a bit steep for a static website, bacause of this the actual blog is accessible through github pages. 

Ideally we'd move away from both and have the blog on our own infrastructure. 

The flow would be something like:

- Create a pr, have it deployed temporarily for people to have a look at.
- Have someone else do the final review and merge into development/master
- [Do we need staging if we use the pr apps for reviewing?]
- Automatically deploy to xxx once the pr is merged into master
